src/IB/IBFEMethod.cpp

```diff
@@ -1054,7 +1138,14 @@ IBFEMethod::computeLagrangianForce(const double data_time)
         {
             computeStressNormalization(*d_Phi_half_vecs[part], *d_X_half_vecs[part], data_time, part);
         }
+        // Zero-out force vector
+        d_F_half_vecs[part]->zero();
```

Comment:
Does this need to be zero-ed out here?

Reply:
That seemed to me the local place to make this change. Other place is in the preprocessIntegrateHierarchy(), but that seemed non-local spot for force calculation. What do you think?

Reply:
I think that computeInteriorForceDensity is already overwriting the F vector; in that case, the call to zero() is not needed.